src/lsst/cst/conversions/tools.py

```diff
@@ -51,214 +50,93 @@ def data_id_to_str(data_id: dict):
     Returns
     -------
     data_id_str: `str`
-        Data identifier string
+        Data identifier string.
     """
-    cal_exp_id = CalExpId(**data_id)
-    return str(cal_exp_id)
+    data_id_str = json.dumps(data_id)
+    return data_id_str
 
 
-def sort_dataframe(
-    df: pd.DataFrame,
-    sort_key: str,
-    ascending: bool = False,
-    set_index: bool = True,
-) -> pd.DataFrame:
-    """Return a sorted copy of the dataframe 'df' by index,
-    selecting the desired order (ascending or descending)
-    using the ascending argument, it also exchanges the index
-    of the dataframe for the sort_key if set_index parameter
-    is True (default).
+def psf_size_at_pixel_xy(psf, bbox, xy: (int, int)) -> dict:
```

Comment:
```suggestion
def psf_size_at_pixel_xy(psf, bbox, xy: tuple[int, int]) -> dict[str, float]:
```
The reason the linter doesn't let you annotate psf and bbox is because those types aren't imported at the top of the file (which you can do with a `TYPE_CHECKING` block if you like).

Reply:
Made the change to `def psf_size_at_pixel_xy(psf, bbox, xy: tuple[int, int]) -> dict[str, float]`.

That makes sense for why the linter complains, thanks. I'll leave them unimported b/c they'd have to be in a try/except anyway.